front/src/components/UI/Tabs/EditTabs/EditTabs.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { PackageData } from '../../../App/Help/Help';
+import { EditTabsButtonStyled, EditTabsInputStyled, EditTabsInputWideStyled} from './EditTabs.styled';
+
+const EditTabs = (props: any) => {
+  //destructure the element
+  const [state, setState] = useState<PackageData>(props.location.state.element);
+
+  const updateElement = (e: any) => {
+    e.preventDefault();
+
+    if (state.name === "" || state.content === "") {
+      alert("Wypelnij wszystkie pola!");
+      return;
+    }
+
+    props.updateElementHandler(state);
+    props.history.push("/help");
+
+  };
+  return (
+    <div>
+      <form onSubmit={updateElement}>
+        Name:
+        <EditTabsInputStyled type="text" 
```

Comment:
<EditTabsInputStyled 
type="text"

Jeśli jest dużo propsów przekazywanych to każdy robimy w nowej lini

Reply:
Dodatkowo brakuje wcięć (pomoże użycie ALT+ SHIFT + F w VSCode)